### PR TITLE
N interactive list

### PR DESCRIPTION
**Pull request by Chike1990**
In this milestone, I used ES6 import and export statements to import modules as I add more interactive features to the to-do app.

### CODE REVIEW

**Comment by mikenath223** [comment]
Hi @Chike1990 , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Kindly remove `node_modules` from committed files and include it in `.gitignore`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

File: node_modules/.bin/cssesc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
+
+case `uname` in
+    *CYGWIN*|*MINGW*|*MSYS*) basedir=`cygpath -w "$basedir"`;;
```
Comment (Davidosky007) [inline node_modules/.bin/cssesc]: *  I strongly suggest you remove node modules from Github
* it is not best practice 
* you can run this in the terminal `git rm -r --cached . |
git add . |
git commit -m "remove gitignore files" |
git push` 
image

File: dist/index.html [hunk 2]
```diff
@@ -1,12 +1,40 @@
...
+    </div>
+
+    <div class="todo__items">
+      <!-- <div class="todo__placeholder">
+       <div>
+         <input type="checkbox" class="todo__checkbox" >
+         <input class="todo__text" value = "Learn Coding"/>
+       </div>
+        <i class="todo__drag-icon fas fa-ellipsis-v" ></i>
+      </div> -->
```
Comment (Davidosky007) [inline dist/index.html]: * Kindly remove commented code from your codebase  it makes your codebase bulky and unprofessional

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Chike1990 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: node_modules/.bin/cssesc (hunk 1 above)
Comment (rloterh) [reply node_modules/.bin/cssesc]: This issue is partially well resolved.  
- [ ] OPTIONAL
Nice job for removing the node modules as recommended. However, kindly also endeavor to exclude the `.vscode` from your codebase by including it's path in the _.gitignore_

File: dist/index.html (hunk 2 above)
Comment (rloterh) [reply dist/index.html]: This issue is not properly resolved.

- [ ] Kindly endeavor to remove the comments as informed by the previous reviewer.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Chibuike, 

You did great on this project, and for making the changes requested by the previous reviewer!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [ ] _Kindy check the comments above, made under the reviews of the previous code reviewer(s) and stating **not properly resolved**. Kindly take your time to correctly review those issues outlined by the previous code reviewers, and please check to see that the changes are working as intended._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider the comments marked [OPTIONAL] under this review:
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,53 @@
-# TodoList
\ No newline at end of file
+# ToDo List App
+
+Todo List is a web application that allows users to add/remove the list of "to-dos". It provides an interface for adding to do activities, for visualizing the list of added tasks to complete. This is achieved using plain HTML, CSS, JavaScript objects, arrays, DOM, basic events, and webpack.
+
+![Screenshot](src/Screenshot.png)
```
Comment (rloterh) [inline README.md:5]: - [ ]  OPTIONAL
This _screenshot_ link is broken. Kindly endeavor to fix it

**Review by rloterh — CHANGES_REQUESTED** [review]
## Status: Approved :heavy_check_mark:

Hi Chibuike, 

Your project is complete! Great job for making the necessary changes recommended in the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked [OPTIONAL] under this review:_



<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (Chike1990) [reply README.md:5]: Ok, thank you so much

**Pull request merged**